### PR TITLE
Delegate tokens page in KEEP token dashboard

### PR DESCRIPTION
**Pull request by r-czajkowski**
Closes: #1634 

This PR updates a Tokens page UI according to the Figma view. A PR doesn't provide icons in buttons and doesn't display a token balance with suffixes and KEEP icon eg. 20M. It will be resolved in followup PR.

<details>
  <summary>Screenshots</summary>

obraz

obraz

</details>

### CODE REVIEW

File: solidity/dashboard/src/contexts/TokensPageContext.jsx [hunk 1]
```diff
@@ -15,7 +15,11 @@ import tokensPageReducer, {
   SET_STATE,
   GRANT_STAKED,
   GRANT_WITHDRAWN,
+  SET_SELECTED_GRANT,
 } from "../reducers/tokens-page.reducer"
+import { isEmptyObj } from "../utils/general.utils"
+import { findIndexAndObject } from "../utils/array.utils"
+import { usePrevious } from "../hooks/usePrevious"
 
 const tokesnPageServiceInitialData = {
```
Comment (dimpar) [inline solidity/dashboard/src/contexts/TokensPageContext.jsx:24]: typo, tokensPage...

File: solidity/dashboard/src/components/DelegationOverview.jsx [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+  const getUndelegations = () => {
+    if (tokensContext === "granted") {
+      return grantUndelegations
+    }
+
+    return ownedUndelegations
+  }
+
+  return (
+    <section>
```
[the remark this answers is not in the document]
Comment (r-czajkowski) [reply solidity/dashboard/src/components/DelegationOverview.jsx:58]: It's a 'special' type of section and using the `<Tile>` component would be too complicated.

File: solidity/dashboard/src/components/OwnedTokensOverview.jsx [hunk 3]
```diff
@@ -0,0 +1,44 @@
+import React, { useMemo } from "react"
+import Tile from "./Tile"
+import { CircularProgressBars } from "./CircularProgressBar"
+import { add } from "../utils/arithmetics.utils"
+import { displayAmount } from "../utils/general.utils"
+import { colors } from "../constants/colors"
+
+const OwnedTokensOverview = ({ keepBalance, stakedBalance }) => {
```
[the remark this answers is not in the document]
Comment (r-czajkowski) [reply solidity/dashboard/src/components/OwnedTokensOverview.jsx:8]: Hmm... I think it's related to our internal testnet. I'm pretty sure it should work on `ropsten` and `mainnet`. I'll double-check.

**Comment by dimpar** [comment]
Something is wrong with "Token Owned" view

Screenshot 2020-05-04 at 11 21 37

File: solidity/dashboard/src/contexts/TokensPageContext.jsx [hunk 4]
```diff
@@ -21,7 +21,7 @@ import { isEmptyObj } from "../utils/general.utils"
 import { findIndexAndObject } from "../utils/array.utils"
 import { usePrevious } from "../hooks/usePrevious"
 
-const tokesnPageServiceInitialData = {
+const tokennsPageServiceInitialData = {
```
Comment (dimpar) [inline solidity/dashboard/src/contexts/TokensPageContext.jsx:24]: tokennsPageServiceInitialData -> tokensPageServiceInitialData

**Pull request merged**